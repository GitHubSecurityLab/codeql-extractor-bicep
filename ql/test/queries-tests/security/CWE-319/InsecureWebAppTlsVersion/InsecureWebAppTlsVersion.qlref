security/CWE-319/InsecureWebAppTlsVersion.ql
