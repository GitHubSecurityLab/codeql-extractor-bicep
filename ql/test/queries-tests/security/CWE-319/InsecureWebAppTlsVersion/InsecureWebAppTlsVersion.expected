| app.bicep:15:1:23:1 | AppService[webAppBad1] | Azure Web App configured with insecure TLS version: 1.1 |
| app.bicep:26:1:34:1 | AppService[webAppBad2] | Azure Web App configured with insecure TLS version: 1.0 |
